public/404.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Single Page Apps for GitHub Pages</title>
```

Comment:
Should this be changed to something more descriptive?

Reply:
I just changed it to match `index.html`